docs/releases/release-v2.6.md

```diff
@@ -0,0 +1,94 @@
+# Enterprise Distribution for Flux v2.6.x
+
+- [k8s](#supported-kubernetes-versions)
+- [apis](#api-versions)
+  - [ga](#general-availability-ga)
+  - [beta](#beta-preview)
+- [v2.6.0](#v260)
+  - [mainline](#mainline-v260)
+  - [FIPS-compliant](#fips-compliant-v260)
+
+## Supported Kubernetes Versions
+
+| Distribution | Versions                                                   |
+|:-------------|:-----------------------------------------------------------|
+| Kubernetes   | 1.27 <br>1.28 <br>1.29 <br>1.30 <br>1.31 <br>1.32 <br>1.33 |
+| OpenShift    | 4.12 <br>4.13 <br>4.14 <br>4.15 <br>4.16 <br>4.17 <br>4.18 |
```

Comment:
Are these correct?

Reply:
I’ll update these after running all the tests